README.md

```diff
@@ -51,7 +51,31 @@ wskdeploy
 ```
 
 ### Bind Service Credentials
-You will need to bind your service credentials to the package, so that the Actions will have access to the service credentials. Individual README instructions within each package detail this step.
+You will need to bind your service credentials to the package, so that the Actions will have access to the service credentials. Individual README instructions within each package detail this step with specific commands to use per package.
+
+Binding credentials is the preferred way to authenticate actions. To do so, it will follow the following pattern.
+
+```
+bx wsk service bind <name-of-service> <name-of-package>
+```
+
+### Passing Authentication Parameters
+
+You may optionally pass in authentication parameters to override the service credential bind as paramaters to each action.
+
+- In some instances, you authenticate by providing the `username` and `password` for the service instance.
+- Visual Recognition uses a form of `api_key` only with instances created before May 23, 2018. Newer instances of Visual Recognition use [IAM](#iam).
```

Comment:
I wouldn't put the Visual Recognition stuff here. But it would be good to have details in the specific package README

BTW, you might want to standardize on either `-` or `*` for list items just for consistency.